src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands/NewAzureRmAnalysisServicesServer.cs

```diff
@@ -92,7 +97,13 @@ public override void ExecuteCmdlet()
                     }
                 }
 
-                var createdServer = AnalysisServicesClient.CreateOrUpdateServer(ResourceGroupName, Name, Location, Sku, Tag, Administrator);
+                var availableSkus = AnalysisServicesClient.ListSkusForNew();
```

Comment:
@taiwu out of curiosity, why replace the `ValidateSet` with the logic below? Do they not accomplish the same thing? Using a `ValidateSet` was a better experience for users as they were able to tab through the possible values; this extra functionality is now unavailable to the user.